main/menu.ts

```diff
@@ -20,14 +20,28 @@ export default function setMenu(win: Electron.BrowserWindow) {
                     label: 'Undo',
                     accelerator: 'CmdOrCtrl+Z',
                     click: () => {
-                        win.webContents.send('nyaovim:exec-commands', ['undo']);
+                        const focusedWebContents = webContents.getFocusedWebContents();
+                        if (win.webContents === focusedWebContents) {
```

Comment:
I can get the `isFocused()` in the dev tool by `require('electron').remote.getCurrentWindow().webContents.isFocused()`.

However, `tsc` reports `isFocused` doesn't exist. 
![2016-11-30 5 11 11](https://cloud.githubusercontent.com/assets/4080524/20746260/067efdb8-b720-11e6-85d4-87d5321dd646.jpg)

Can you tell me what's wrong?